src/js/plugins/bluetooth.js

```diff
@@ -210,86 +230,83 @@ class Bluetooth extends Plugin {
 
     scanForDevices() {
         this.status(`Start scanning`);
-        api.putPlugin(this.callsign, 'Scan', null, (err, resp) => {
-            if (err !== null) {
-                console.error(err);
-                return;
-            }
-
-            // update after 2s
-            setTimeout(this.update.bind(this), 2000);
-
-            // update discovered device list in every 4s
-            this.Timer = setInterval(this.getDiscoveredDevices.bind(this), 4000);
+        var f = document.getElementById("BT_DeviceType");
+        var device = f.options[f.selectedIndex].value;
 
-            this.status(`Scanning...`);
-            // stop scan after 1 min
-            setTimeout(this.stopScan.bind(this), 60000);
-
-        });
-    }
-
-    stopScan() {
-        this.status(`Stopping Scan`);
-        clearInterval(this.Timer);
-        api.putPlugin(this.callsign, 'StopScan', null, (err, resp) => {
+        api.req('PUT', api.getURLStart('http') + this.callsign +'/Scan/?LowEnergy='+device,
+                     null,BluetoothControl.1.status,{}, (err, resp) => {
```

Comment:
I guess `BluetoothControl.1.status` should be a string? `'BluetoothControl.1.status'`?